s3_resource/include/irods/private/s3_resource/multipart_shared_data.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef S3_RESOURCE__MULTIPART_SHARED_DATA_HPP
```

Comment:
Collapse double underscores to single underscore.

Reply:
done